### ISSUE TITLE
Installation with Python 3.5 failed

### ISSUE THREAD

**Issue by LJBD** [vxgmichel/python-tango-devicedoc#1]
Hey!

I've tried installing devicedoc in a conda environment with Python 3.5 and it failed. Here is the log:
```python
(pytango_workshop) [tango@solaris-dev source]$ pip install git+https://github.com/vxgmichel/pytango-devicedoc
Collecting git+https://github.com/vxgmichel/pytango-devicedoc
  Cloning https://github.com/vxgmichel/pytango-devicedoc to /tmp/pip-iqhplsci-build
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-iqhplsci-build/setup.py", line 9, in <module>
        include_package_data=True,
      File "/home/tango/miniconda3/envs/pytango_workshop/lib/python3.5/distutils/core.py", line 121, in setup
        dist.parse_config_files()
      File "/home/tango/miniconda3/envs/pytango_workshop/lib/python3.5/distutils/dist.py", line 402, in parse_config_files
        val = parser.get(section,opt)
      File "/home/tango/miniconda3/envs/pytango_workshop/lib/python3.5/configparser.py", line 797, in get
        d)
      File "/home/tango/miniconda3/envs/pytango_workshop/lib/python3.5/configparser.py", line 393, in before_get
        self._interpolate_some(parser, option, L, value, section, defaults, 1)
      File "/home/tango/miniconda3/envs/pytango_workshop/lib/python3.5/configparser.py", line 443, in _interpolate_some
        "found: %r" % (rest,))
    configparser.InterpolationSyntaxError: '%' must be followed by '%' or '(', found: '%{?dist}.maxlab'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-iqhplsci-build/

```

For Python 2.7 there's no problem.

**Comment by vxgmichel** [comment]
Fixed: c492b99
Thanks!

**Comment by LJBD** [comment]
Great, thanks!
